README.rst

```diff
@@ -142,6 +141,40 @@ https://devdocs.line.me/en/#bot-api-get-profile
     print(profile.picture_url)
     print(profile.status_message)
 
+get\_group\_member\_profile(self, group\_id, user\_id, timeout=None)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Gets the user profile of a member of a group that
+    the bot is in. This can be the user ID of a user who has
```

Comment:
Could you remove unneeded space ?
It seems that rendering is failed.

<img width="988" alt="2017-08-15 13 26 42" src="https://user-images.githubusercontent.com/903482/29302292-d43497be-81bd-11e7-93f0-a198ee459308.png">